iOSUnitTests/AuthorityListViewModelTests.swift

```diff
@@ -0,0 +1,69 @@
+//
```

Comment:
Added some basic test coverage to encourage applicants to discuss with interviewers what tests they would add if they had more time.